docs/explore/redisinsightv2/windows/index-windows.mdx

```diff
@@ -0,0 +1,173 @@
+
+:::tip TIP
+For a limited time, use **TIGER200** to get **$200** credits on Redis Enterprise Cloud and try all the advanced capabilities!
+
+:tada: [Click here to sign up](https://redis.com/try-free)
+
+:::
+
+![endpoint of redis enterprise cloud](images/rediscloud_endpoint.png "image_tooltip")
+
+### Step 2. Download and Install RedisInsight
+
+
+To use RedisInsight on a local Windows, you can download the software directly from the Redis official website:
+
+[Click this link](https://redis.com/redis-enterprise/redis-insight/#insight-form) to access a form that allows you to select the operating system of your choice(in our case, it's Microsoft Windows).
+
+
+![form to fill to download RedisInsight](images/redisinsight-form.png)
+
+Fill out the rest of the form and click “Download”.  
+
+#### Install RedisInsight
+
+Click on the RedisInsight executable(in the form of .exe) and install it in your system.
+
+![setup redisinsight](images/redisinsight-setup.png)
+
+Once the RedisInsight software is installed, click on its icon to open the RedisInsight application. It will display the End-User License Agreement and Privacy Settings. Enable Analytics and Encrypt sensitive information as per your preference.
+
+![accept redisinsigh licence](images/redisinsight-eula.png)
+
+### Step 3. Connect to a Redis Database
+
+Enter the requested details, including Host (endpoint), Port, and Alias in the form, as shown below. You can use "default" as username for now. Then click “ADD REDIS DATABASE”.
+
+![adding redis database](images/redisinsight-add-database.png)
+
+Once added, you will see the database name listed as shown below:
+
+![listing the redis database](images/redisinsight-db-added.png)
+
+### Step 4. Use "Browser Tool"
+
+Click on the "Key" icon on the left sidebar to open up the browser tool.
+
+![redis database with no keys](images/redisinsight-no-keys.png "image_tooltip")
+
+
+### Step 5. Overview of User database keys
+
+Let us import a user database (6k keys). This dataset contains users stored as Redis Hashes.
+
+
+### 
+**Users**
+
+The user hashes contain the following fields:
+
+
+
+*   `user:id` : The key of the hash.
+*   `first_name` : First Name.
+*   `last_name` : Last name.
+*   `email` : email address.
+*   `gender` : Gender (male/female).
+*   `ip_address` : IP address.
+*   `country` : Country Name.
+*   `country_code` : Country Code.
+*   `city` : City of the user.
+*   `longitude` : Longitude of the user.
+*   `latitude` : Latitude of the user.
+*   `last_login` : Epoch time of the last login.
+
+### Step 6. Clone the repository
+
+Open up the CLI terminal and run the following command:
+
+ ```bash
+  git clone https://github.com/redis-developer/redis-datasets
+  cd redis-datasets/user-database
+ ```
+
+
+### Step 7. Import the user database keys
+
+Open up the CLI terminal and run the following command:
```

Comment:
They'll also need a password to do this for a cloud database, and maybe a reminder to use the host and port of their cloud database instance?

Reply:
Fixed.